eZ/Publish/API/Repository/Tests/FieldType/DateIntegrationTest.php

```diff
@@ -339,4 +339,38 @@ public function providerForTestIsNotEmptyValue()
             ),
         );
     }
+
+    protected function getValidSearchValueOne()
+    {
+        $date = new DateTime( "1970-01-02" );
+        return $date->getTimestamp();
+    }
+
+    protected function getValidSearchValueTwo()
+    {
+        $date = new DateTime( "1970-01-03" );
+        return $date->getTimestamp();
+    }
+
+    protected function getSearchTargetValueOne()
+    {
+        // Handling Legacy Search Engine, which stores Checkbox value as integer
```

Comment:
Fixed in https://github.com/ezsystems/ezpublish-kernel/commit/af727a7f723c214d7edf081c7a7fb15cc33ed4e3.